python/cli/prelude_cli/views/build.py

```diff
@@ -42,22 +42,41 @@ def create_test(controller, name, unit, test, techniques, advisory):
         )
 
     if not test:
+        """ Mark-up, upload, and write the test file template. """
         basename = f'{t["id"]}.go'
-        template = pkg_resources.read_text(templates, 'template.go')
-        template = template.replace('$ID', t['id'])
-        template = template.replace('$NAME', name)
-        template = template.replace('$UNIT', unit or '')
-        template = template.replace('$CREATED', str(datetime.utcnow()))
+        utc_time = str(datetime.utcnow())
```

Comment:
`utcnow` is on it's way out, we started removing it from the backend code. Since you're touching this, maybe make it timezone aware while you're at it? 

* https://blog.miguelgrinberg.com/post/it-s-time-for-a-change-datetime-utcnow-is-now-deprecated
* example of a tz aware datetime: https://github.com/preludeorg/backbone/blob/5a595f551202d1e932c52f815954670c1e69ae86/backbone/objects/standard.py#L73C18-L73C18